### ISSUE TITLE
Is Tabnine abandoned?

### ISSUE THREAD

**Issue by drcongo** [codota/tabnine-sublime#118]
- OS version: macOS Monterey 12.3
 - Editor version: SublimeText 4 (4126)
 - Programming language: any
 - Tabnine extension version: 3.2.0

SublimeText 4 has been out for almost a year now, and Tabnine is still unusable with it. The last commit in the `task/support-sublime-4` branch was 9 months ago. Is there any hope of Tabnine ever being usable again?

**Comment by dimacodota** [comment]
Hi @drcongo , Tabnine does supports ST4, can you please try the following :
 - open tabnine seetings ` Preferences -> Package Settings -> tabnine -> Settings - User`
 - set the `native_auto_complete` property to `true`

**Comment by drcongo** [comment]
Hi @dimacodota, thanks, that's got me part of the way there, but I'm still unable to tab through the results - regardless of whether `auto_complete_commit_on_tab` in Sublime's settings is set to `true` or `false`, hitting tab commits the first result, and the first result is never a Tabnine result. There used to be excellent documentation on the Tabnine site for setting up Sublime to work with Tabnine and now there's none at all beyond ["Install Tabnine"](https://www.tabnine.com/install/sublime). There's no mention of setting `native_auto_complete` to `true` or what settings need to be changed in Sublime's own settings. The only extra info available is at the bottom of the README about settings `auto_complete` to `false`, which actually disables Tabnine's completions too so appears to actually be wrong.

**Comment by KishCom** [comment]
Yes, this is plugin is about as abandoned as you can get without _actually_ abandoning a plugin.

It's [clear as day](https://github.com/codota/tabnine-sublime/issues) that no one at Tabnine uses Sublime Text but they wanted to be able to put it under the list of IDEs they "support".

**Comment by Gabriel-p** [comment]
Today I checked and Sublime Text is not even listed as supported anymore: https://www.tabnine.com/install/

**Comment by Eric-Guo** [comment]
I just change my MBP and found this issue, at least TabNine works again in local mode and I just write a note which I think may helpful for other TabNine and Sublime Text user who having API Key.

https://blog.cloud-mes.com/2025/04/09/make-tabnine-continue-work-after-official-stop-support-the-sublime-text/

**Comment by rgpublic** [comment]
This is all really sad. I'm currently investigating AI options for our company, but without Sublime support this is basically a non-starter. Anyone knows why they abandoned this? Sublime is quite a popular editor.

**Comment by Eric-Guo** [comment]
@rgpublic Sublime is relatively less popular, than zed, I guess? 🤣 

I'm currently using cursor, the auto tab is really good, also very similar to Sublime with Tier 1 agent support.

**Comment by rgpublic** [comment]
@Eric-Guo : Thanks I know about them, of course. But the whole company is using the same editor; a lot of Sublime plugins; specific settings etc. That would all need to be switched over. Completely unrealistic.
